docs/Reference/Pantheon-CLI-Syntax.md

```diff
@@ -307,6 +307,27 @@ max-peers=42
 Specifies the maximum P2P peer connections that can be established.
 The default is 25.
 
+### max-remote-connections-percentage
```

Comment:
This name should be updated since we just merged: https://github.com/PegaSysEng/pantheon/pull/1715 .

Reply:
Done